test/GasConsumptionKovan.js

```diff
@@ -0,0 +1,308 @@
+  
+        //await erc20FeeProxy.setSafeTransferRequired(USDT.address,true);
+  
+        
+        //await USDC.approve(erc20FeeProxy.address,ethers.utils.parseEther("1000"));
+        //await GUSD.approve(erc20FeeProxy.address,ethers.utils.parseEther("1000"));
+        //await USDT.approve(erc20FeeProxy.address,ethers.utils.parseEther("1000"));
+  
+      });
+
+      /*it("Biconomy Forwarder Only Personal", async function(){
+          //new nonce x10 personal
+          //new batch x10 personal
+          for(i=0;i<10;i++){
+            const req = await testRecipient.populateTransaction.nada();
+            req.from = await accounts[1].getAddress();
+            req.batchNonce = i;
+            req.batchId = 0;
+            req.txGas = (req.gasLimit).toNumber();
+            req.tokenGasPrice = 0;
+            req.deadline = 0;
+            delete req.gasPrice;
+            delete req.gasLimit;
+            delete req.chainId;
+            req.token = testnetDai.address;
+            const hashToSign = abi.soliditySHA3(['address','address','address','uint256','uint256','uint256','uint256','uint256','bytes32'],
+                                                [req.from,req.to,req.token,req.txGas,req.tokenGasPrice,req.batchId,req.batchNonce,req.deadline,
+                                                    ethers.utils.keccak256(req.data)]);
+            const sig = await accounts[1].signMessage(hashToSign);
+            const tx = await forwarder.executePersonalSign(req,sig);
+            const receipt = await tx.wait();
+            console.log("Biconomy Forwarder nonce="+req.batchNonce+" gas used :"+receipt.gasUsed.toString()+" Tx Hash :"+receipt.transactionHash)
+          }
+          for(i=0;i<10;i++){
+            const req = await testRecipient.populateTransaction.nada();
+            req.from = await accounts[1].getAddress();
+            req.batchNonce = 0;
+            req.batchId = i+1;
+            req.txGas = (req.gasLimit).toNumber();
+            req.tokenGasPrice = 0;
+            req.deadline = 0;
+            delete req.gasPrice;
+            delete req.gasLimit;
+            delete req.chainId;
+            req.token = testnetDai.address;
+            const hashToSign = abi.soliditySHA3(['address','address','address','uint256','uint256','uint256','uint256','uint256','bytes32'],
+                                                [req.from,req.to,req.token,req.txGas,req.tokenGasPrice,req.batchId,req.batchNonce,req.deadline,
+                                                    ethers.utils.keccak256(req.data)]);
+            const sig = await accounts[1].signMessage(hashToSign);
+            const tx = await forwarder.executePersonalSign(req,sig);
+            const receipt = await tx.wait();
+            console.log("Biconomy Forwarder Batch="+req.batchId+" gas used :"+receipt.gasUsed.toString()+" Tx Hash :"+receipt.transactionHash)
+          }
+      });
+
+      it("Biconomy Forwarder Only EIP712", async function(){
```

Comment:
Are these lines supposed to be commented?

Reply:
need to check with Kiran once which issues he faced in kovan deployment. Btw we probably wont be needing kovan now under tests